python3/emailmessage/README.md

```diff
@@ -0,0 +1,96 @@
+# Email Message
+
+This page describes the methods and fields available for use on the object type `Email`. Each method has a simple example of how to use it.
+
+## Methods
+<blockquote>
+<!-- Start of section -->
+  <!-- Start of method -->
+  <details><summary> getBodyHtmlRaw() </summary>
+
+  Returns the content of the email message, including the HTML markup. The returned string is a unicode string.
+
+  Example:
+  ```python
+  from bs4 import BeautifulSoup
+
+  soup = BeautifulSoup(emailmessage.getBodyHtmlRaw(), "html.parser")
+  log.info(soup.prettify())
```

Comment:
The problem with prettify is that hardly any of our fields that take text respect formatting. We might give false hope here.